iree/compiler/Dialect/HAL/IR/HALOps.td

```diff
@@ -1858,6 +1879,37 @@ def HAL_ExecutableEntryPointOp : HAL_Op<"executable.entry_point", [
     TypeAttr:$signature,
     OptionalAttr<HAL_WorkgroupSizeAttr>:$workgroup_size
   );
+
+  let regions = (region VariadicRegion<SizedRegion<1>>:$num_workgroups_region);
```

Comment:
naming: let's stick to `workgroup_count` - no `num_xx` - so that we are consistent here with all the other ops/attrs.